tests/inputs/twiss_optics/readme.txt

```diff
@@ -0,0 +1,6 @@
+Twiss Files are from a default LHC lattice with assigned a2b2-a6b6 errors
+and crossing scheme.
+
+The lattice is not sliced and hence will not work for RDT checks!
+To compute comparable RDTs in crossing scenario to PTC-RDT calculations,
+one needs a sliced lattice!
```

Comment:
Here you mean the number of integration steps in PTC `nst`?

Reply:
No. I mean if you use a thick lattice, the analytical calculation will not be exact enough. I'll rephrase